plugins/woocommerce/client/legacy/js/admin/settings-views-html-settings-tax.js

```diff
@@ -19,6 +19,7 @@
 			$tbody             = $( '#rates' ),
 			$save_button       = $( ':input[name="save"]' ),
 			$pagination        = $( '#rates-pagination' ),
+			$bottom_pagination = $( '#rates-bottom-pagination' ),
```

Comment:
@TarekNabil Thanks for the PR. Is there a reason you didn't include the new pagination element in `$pagination`?

```
$pagination = $( '#rates-pagination, #rates-bottom-pagination' )
```

Doing that would eliminate all the other JS changes.

Reply:
What do you think I should do now? Make the edits and commit again or create a new PR?